packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -726,19 +726,25 @@ export default class Meetings extends WebexPlugin {
    * @memberof Meetings
    */
   public register() {
-    Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.MEETINGS_REGISTRATION_STEP, {
-      step: '[sdk] begin registration',
-    });
+    const submitStepMetric = (step) => {
+      try {
+        Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.MEETINGS_REGISTRATION_STEP, {
```

Comment:
Are these metrics being sent only temporarily to debug some issue and will be removed in a few days or are they to stay? I'm asking because there are a lot of them - 3 for each time any user loads our app. There was almost a million of them sent yesterday and we have a very tight budget on the number of amplitude events we can send.

Also the event "meetings_registration_step" doesn't have the "js_sdk_" prefix which all other SDK amplitude events have, but I guess it's not a big deal especially if they are only temporary.

Reply:
Didn't realize we had a tight budget for amplitude metrics.  I've changed the PR to remove them, instead.  This won't do in until Tuesday at the earliest, so there should be enough data for debugging.